zerver/openapi/zulip.yaml

```diff
@@ -11746,6 +11746,194 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /invites:
+    get:
+      operationId: get-invites
+      summary: Get all invites
+      tags: ["invites"]
+      description: |
+        Fetch all the invitations (i.e. email invitations and reusable invitation
+        links) that can be controlled by the user.
+
+        This isn't necessarily the same as all the invitations generated by the
+        user, as administrators can also control invitations that they did not
+        themselves create.
+
+        **Changes**: Prior to Zulip 8.0 (feature level 209), non-admin users could
+        only create email invitations, and therefore the response did not include
+        reusable invitation links for these users.
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      invites:
+                        type: array
+                        description: |
+                          An array of `invites` object.
+                        items:
+                          $ref: "#/components/schemas/Invite"
+                    example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "invites":
+                          [
+                            {
+                              email: "example@zulip.com",
+                              expiry_date: null,
+                              id: 1,
+                              invited: 1710606654,
+                              invited_as: 200,
+                              invited_by_user_id: 9,
+                              is_multiuse: false,
+                            },
+                            {
+                              expiry_date: 1711463862,
+                              id: 1,
+                              invited: 1710599862,
+                              invited_as: 400,
+                              invited_by_user_id: 9,
+                              is_multiuse: true,
+                              link_url: "http://localhost:9991/join/6i4324pzbtyvmwv5nwyghoof/",
+                            },
+                          ],
+                      }
+    post:
+      operationId: send-invites
+      summary: Send invites
+      tags: ["invites"]
+      description: |
+        Send invitations to the email addresses of the invitees.
```

Comment:
You should link to https://zulip.com/help/invite-new-users from this description; probably from the `GET` description as well.